### ISSUE TITLE
Unexpected behavior for chromoqc chrom.s and chrom.e arguments

### ISSUE THREAD

**Issue by zkamvar** [knausb/vcfR#61]
I was attempting to zoom into a region with chromoqc using the chrom.s and chrom.e arguments when I came upon some unexpected behavior (see reproducible example below):

1. When supplying `chrom.s`, the first three rows of the visual shift, but the last four rows don't
2. When supplying `chrom.e`, I get an error.

I am using the current master version of vcfR (see the session info below).

``` r
# made with reprex version 0.1.1
library("vcfR")
#> 
#>    *****       ***   vcfR   ***       *****
#>    This is vcfR 1.4.0.9000 
#>      browseVignettes('vcfR') # Documentation
#>      citation('vcfR') # Citation
#>    *****       *****      *****       *****
data(vcfR_example)
chrom <- create.chromR(name = "Supercontig", vcf = vcf, seq = dna, ann = gff, 
  verbose = FALSE)
chrom <- masker(chrom, min_QUAL = 0, min_DP = 350, max_DP = 650, min_MQ = 59.5, 
  max_MQ = 60.5)
chrom <- proc.chromR(chrom, verbose = FALSE)
chromoqc(chrom, dp.alpha = 22)
```

![](http://i.imgur.com/g53Pb1F.png)

``` r
chromoqc(chrom, dp.alpha = 22, chrom.s = 20000)
```

![](http://i.imgur.com/CplRvyt.png)

``` r
chromoqc(chrom, dp.alpha = 22, chrom.s = 20000, chrom.e = 40000)
#> Error in chromo(chrom, boxp = boxp, chrom.e = chrom@len, drlist1 = myList1, : formal argument "chrom.e" matched by multiple actual arguments
```

<details><summary>Session info</summary>

``` r
devtools::session_info()
#> Session info --------------------------------------------------------------
#>  setting  value                       
#>  version  R version 3.3.2 (2016-10-31)
#>  system   x86_64, darwin13.4.0        
#>  ui       X11                         
#>  language (EN)                        
#>  collate  en_US.UTF-8                 
#>  tz       America/Chicago             
#>  date     2017-02-27
#> Packages ------------------------------------------------------------------
#>  package     * version    date       source                            
#>  ape           4.1        2017-02-14 CRAN (R 3.3.2)                    
#>  backports     1.0.5      2017-01-18 CRAN (R 3.3.2)                    
#>  bitops        1.0-6      2013-08-17 CRAN (R 3.2.0)                    
#>  cluster       2.0.5      2016-10-08 CRAN (R 3.3.0)                    
#>  devtools      1.12.0     2016-06-24 CRAN (R 3.3.0)                    
#>  digest        0.6.12     2017-01-27 CRAN (R 3.3.2)                    
#>  evaluate      0.10       2016-10-11 cran (@0.10)                      
#>  formatR       1.4        2016-05-09 CRAN (R 3.3.0)                    
#>  htmltools     0.3.5      2016-03-21 CRAN (R 3.2.4)                    
#>  knitr         1.15.11    2017-02-21 Github (yihui/knitr@806e171)      
#>  lattice       0.20-34    2016-09-06 CRAN (R 3.3.0)                    
#>  magrittr      1.5        2014-11-22 CRAN (R 3.2.0)                    
#>  MASS          7.3-45     2015-11-10 CRAN (R 3.2.2)                    
#>  Matrix        1.2-8      2017-01-20 CRAN (R 3.3.2)                    
#>  memoise       1.0.0      2016-01-29 CRAN (R 3.2.3)                    
#>  mgcv          1.8-17     2017-02-08 CRAN (R 3.3.2)                    
#>  nlme          3.1-131    2017-02-06 CRAN (R 3.3.2)                    
#>  permute       0.9-4      2016-09-09 cran (@0.9-4)                     
#>  pinfsc50      1.1.0      2016-12-02 CRAN (R 3.3.2)                    
#>  Rcpp          0.12.9     2017-01-14 CRAN (R 3.3.2)                    
#>  RCurl         1.95-4.8   2016-03-01 CRAN (R 3.2.4)                    
#>  rmarkdown     1.3.9003   2017-02-21 Github (rstudio/rmarkdown@cae9df8)
#>  rprojroot     1.2        2017-01-16 CRAN (R 3.3.2)                    
#>  stringi       1.1.2      2016-10-01 CRAN (R 3.3.0)                    
#>  stringr       1.2.0      2017-02-18 cran (@1.2.0)                     
#>  vcfR        * 1.4.0.9000 2017-02-21 Github (knausb/vcfR@fc8d9da)      
#>  vegan         2.4-2      2017-01-17 CRAN (R 3.3.2)                    
#>  viridisLite   0.1.3      2016-03-12 CRAN (R 3.2.4)                    
#>  withr         1.0.2      2016-06-20 cran (@1.0.2)                     
#>  XML           3.98-1.5   2016-11-10 CRAN (R 3.3.2)                    
#>  yaml          2.1.14     2016-11-12 cran (@2.1.14)
```

</details>

**Comment by knausb** [comment]
Hi Zhian,

You've found legacy parameters that I no longer use. I suggest using 'xlim()' because I felt it was more 'R-like.'

I've changed 'chromo()' so that it now throws an error message when non default values for chrom.s or chrom.e are used.

This has made me realize that I have no 'length()' method for chromR objects. I've added this.

These changes have been merged to master and are available at: 9f49557aab14c32d3bdb2f68ee2dcbac971048e4

Thanks and please close if you are satisfied.
Brian

**Comment by knausb** [comment]
Example of use of `xlim()` with `chromo()` can be found here:

https://knausb.github.io/vcfR_documentation/quick_intro.html

**Comment by knausb** [comment]
Addressed at: cee1b88756e581631301ecd7c220e502428da626. Thanks @zkamvar!
